MAINTAINERS.yml

```diff
@@ -3531,7 +3531,7 @@ Silabs Platforms:
     - dts/arm/silabs/
     - dts/bindings/*/silabs*
     - drivers/*/*_gecko*
-    - drivers/bluetooth/hci/slz_hci*
+    - drivers/bluetooth/hci/hci_silabs_*
```

Comment:
You can drop this line since `drivers/*/*_silabs*` also matches it.

Reply:
If it doesn't, I believe we should replace this line by `drivers/*/*/*_silabs*`.

Reply:
Out of scope for this PR, but since HCI drivers are the only kind of HCI driver we have, I think it's worth considering dropping the `hci` subdirectory completely. It'd then also be consistent with the fact that the header file for such drivers is `include/zephyr/drivers/bluetooth.h`.